codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2GoAwayFrame.java

```diff
@@ -87,6 +93,17 @@ public Http2GoAwayFrame setExtraStreamIds(int extraStreamIds) {
     }
 
     @Override
+    public Http2GoAwayFrame setLastStreamId(int lastStreamId) {
+        this.lastStreamId = lastStreamId;
+        return this;
+    }
+
+    @Override
+    public Integer lastStreamId() {
```

Comment:
@normanmaurer yeah the intention was to return `null` if not yet set. Maybe more consistent with other code, if we returned `-1` instead. Hopefully in this code review we will come up with a away to completely remove `lastStreamId` (see https://github.com/netty/netty/pull/5324#discussion_r65067122).